Bucketeer2/Sources/Public/BKTError.swift

```diff
@@ -0,0 +1,87 @@
+import Foundation
+
+public enum BKTError: Error, Equatable {
```

Comment:
I used the named `BKTError` not `BKTException` because of throwing an error in Swift Exception handling.